templates/show_params.mako

```diff
@@ -205,8 +202,51 @@
     ${ render_msg( 'One or more of your original parameters may no longer be valid or displayed properly.', status='warning' ) }
 %endif
 
+
+<h3>Inheritance Chain</h3>
+<div class="inherit" style="background-color: #fff; font-weight:bold;">${hda.name | h}</div>
+
+% for dep in inherit_chain:
+    <div style="font-size: 36px; text-align: center; position: relative; top: 3px">&uarr;</div>
+    <div class="inherit">
+        '${dep[0].name | h}' in ${dep[1]}<br/>
+    </div>
+% endfor
+
+
+
+<h3>Command Line</h3>
+%if job and job.command_line and trans.user_is_admin():
```

Comment:
I'm going to open a PR with these changes - but I think this if should be up a bit so you don't show the heading of an empty block for non-admins.